hca/staging_area_validator.py

```diff
@@ -45,13 +47,16 @@ def __init__(
         staging_area: str,
         ignore_dangling_inputs: bool,
         validate_json: bool,
+        total_retries,
     ) -> None:
         super().__init__()
         self.staging_area = staging_area
         self.validate_json = validate_json
         self.ignore_dangling_inputs = ignore_dangling_inputs
 
         self.gcs = gcs.Client()
+        # Number of retries for validation
+        self.total_retries = total_retries
```

Comment:
To match the existing code there should be a blank line before this and no blank line after.